docs/source/models/gbd2017/causes/lower_respiratory_infections/index.rst

```diff
@@ -46,10 +49,104 @@ Data Description
 
    * - State
      - Definition
```

Comment:
On Friday we discussed adding a "State Name" column to this table. Do people still agree on that?

Reply:
Shouldn't State and State name be synonymous.  We don't have to use single letters in the diagram.

Reply:
I'm open to discussion about it, but I think it is useful to have both an abbreviation/label and a short name. Unless we want to disallow abbreviations and *only* use short names in the diagrams. And the Definition field is a place to further explain the precise meaning of the state in case it's not perfectly clear from the name (so maybe sometimes redundant, but sometimes not).